src/software/ai/navigator/path_planner/theta_star_path_planner.h

```diff
@@ -243,50 +253,66 @@ class ThetaStarPathPlanner : public PathPlanner
      *
      * @return cell in grid
      */
-    Coordinate pointToCoordinate(Point p);
+    Coordinate convertPointToCoord(const Point &p) const;
 
     /**
-     * Try to find a path to destination and leave
+     * Try to find a path to end and leave
      * trail markers along the way
      *
-     * @param dest_coord destination coordinates
+     * @param end_coord end coordinates
      *
-     * @return if path to destination was found
+     * @return true if path to end was found
      */
-    bool findPathToDestination(Coordinate dest_coord);
+    bool findPathToEnd(const Coordinate &end_coord);
 
     /**
-     * Check for invalid or blocked src_coord and dest_coord and
-     * adjust parameters accordingly
+     * Update vertex for all neighbours (all 8 directions) of current_coord
+     *
+     * @param current_coord The current coordinate
+     * @param end_coord end coordinates
+     *
+     * @return true if path to end was found among successors
+     */
+    bool visitSuccessors(const Coordinate &current_coord, const Coordinate &end_coord);
```

Comment:
Might be good to update the function name to use `neighbors` like the comment (I think neighbors is more understandable at first glance, and more common graph/search terminology)

Reply:
changed it